queries/clojure/highlights.scm

```diff
@@ -77,7 +77,8 @@
 
 ; Inline function variables
 ((sym_lit) @variable.builtin
-  (#lua-match? @variable.builtin "^%%"))
+  ; Use #match? to rely on vim's regex for alternations
+  (#match? @variable.builtin "^\\%([0-9][0-9]*|\\&)?$"))
```

Comment:
It's preferable to have multiple patterns instead of a single vim-match (which is _really_ slow) in this case.

Reply:
Good to know, thanks.